AWS High Availability Architecture/README.md

```diff
@@ -0,0 +1,216 @@
+# Introduction
+
+![AWS](https://blogs.bmc.com/wp-content/uploads/2020/06/regions.png)
+
+In this Microbyte we will create `High Availability Architecture` with AWS CLI. First let's understand `what is High Avialability?`
+
+A highly available architecture involves multiple components working together to ensure uninterrupted service during a specific period. This also includes the response time to users’ requests. Namely, available systems have to be not only online, but also responsive.
+
+Implementing a cloud computing architecture that enables this is key to ensuring the continuous operation of critical applications and services. They stay online and responsive even when various component failures occur or when a system is under high stress.
+
+Highly available systems include the capability to recover from unexpected events in the shortest time possible. By moving the processes to backup components, these systems minimize downtime or eliminate it. This usually requires constant maintenance, monitoring, and initial in-depth tests to confirm that there are no weak points.
+
+**Application:**If you want to deploy a website and want's your website to be fast, secure and you want your website to be available all the time even in the time of some failure in the network then`High Availabilty Architecture`comes in play.

```

Comment:
This is seen a bit off, if you want to use bold text here, look at preview once. It looks like this now:
![image](https://user-images.githubusercontent.com/35884252/97731144-cb4e8900-1afa-11eb-842c-f78c3725db35.png)